### PR TITLE
Fix terminal echo on Windows

### PR DESCRIPTION
**Pull request by zanderz**
I started changing the ssl terminal library to have a separate reader and writer, but this is obvs better.
@maxtaco @oconnor663

### CODE REVIEW

File: go/minterm/minterm_nix.go [hunk 1]
```diff
@@ -26,3 +26,7 @@ func (m *MinTerm) open() error {
 	m.width, m.height = w, h
 	return nil
 }
+
+func (m *MinTerm) getReadWriter() io.ReadWriter {
```
Comment (maxtaco) [inline go/minterm/minterm_nix.go]: See C/I, this doesn't build since `io` isn't imported...

**Comment by maxtaco** [comment]
This is cool! :+1: after the compile fix for nix.  Sorry to break everything!

File: go/minterm/minterm_nix.go (hunk 1 above)
Comment (zanderz) [inline go/minterm/minterm_nix.go]: Whoops, thought I could get away with it. Would never merge with such a CI failure anywyaze.

**Comment by oconnor663** [comment]
Neat.

**Pull request merged**